src/bundle/Resources/public/js/scripts/helpers/browser.helper.js

```diff
@@ -0,0 +1,7 @@
+const { userAgent } = window.navigator;
+const isEdg = () => userAgent.includes('Edg');
```

Comment:
We want to use "isEdg" or "isEdge"?

Reply:
Since Edge now have Edg in userAgent. We talked with @GrabowskiM about prople asking if there is type.
If we use isEdg instead of isEdge I think its more logical.